test/extended/deployments/deployments.go

```diff
@@ -304,7 +304,17 @@ var _ = g.Describe("deploymentconfigs", func() {
 				o.Expect(out).To(o.ContainSubstring(fmt.Sprintf("deployment-test-%d up to 1", i+2)))
 				o.Expect(out).To(o.ContainSubstring("--> pre: Success"))
 				o.Expect(out).To(o.ContainSubstring("test pre hook executed"))
-				o.Expect(out).To(o.ContainSubstring("--> Success"))
+
+				// FIXME: In some cases the last line from the deployer pod log is lost so we
+				// can't check the '--> Success' message here.
+				// Instead of relying on the log message, verify the DC condition and check that
+				// the new RC is available. See: origin#13980
+				if dc, _, _, err := deploymentInfo(oc, "deployment-test"); err != nil {
+					o.Expect(err).NotTo(o.HaveOccurred())
+				} else {
+					cond := deployutil.GetDeploymentCondition(dc.Status, deployapi.DeploymentProgressing)
+					o.Expect(cond.Reason).To(o.BeEquivalentTo(deployapi.NewRcAvailableReason))
```

Comment:
@mfojtik ok, since you are using `Reason` the string would differ. 

But I don't think checking for DeploymentProgressing is equivalent to `Success`. Since the tests already check that 'deploymentReachedCompletion' would using `DeploymentAvailable` be a better fit?